lib/ox/document.rb

```diff
@@ -7,14 +7,14 @@ class Document < Element
     #   - _:version_ [String] version, typically '1.0' or '1.1'
     #   - _:encoding_ [String] encoding for the document, currently included but ignored
     #   - _:standalone_ [String] indicates the document is standalone
-    def initialize(prolog={})
+    def initialize(prolog = {})
```

Comment:
I thought we had this changed in Ox to allow no space for default argument values?

Reply:
That has on option of `EnforcedStyle: no_space`, is that the preference for https://docs.rubocop.org/rubocop/cops_layout.html#layoutspacearoundequalsinparameterdefault

Reply:
OK, set the enforced style and reran that rule